changes/changes_worker.go

```diff
@@ -430,27 +430,26 @@ func (w *ChangesWorker) handleHeartbeatStateAndError(changes *exchange.ExchangeC
 }
 
 // Setting up the new polling interval according to the updateType:
-// 	 UPDATE_TYPE_RESET:  set the poll interval to min
-//	 UPDATE_TYPE_ALERT:  set the poll interval to (min + max)/POLL_INTERVAL_ALERT_LEVEL
-//	 UPDATE_TYPE_NEW_CONFIG: adjust the poll interval according to the new configuration from node or node org
-//	 UPDATE_TYPE_NO_CHANGES: increate poll interval because there are no changes
+//   UPDATE_TYPE_RESET:  set the poll interval to min
+//   UPDATE_TYPE_ALERT:  set the poll interval to (min + max)/POLL_INTERVAL_ALERT_LEVEL
+//   UPDATE_TYPE_NEW_CONFIG: adjust the poll interval according to the new configuration from node or node org
+//   UPDATE_TYPE_NO_CHANGES: increate poll interval because there are no changes
 
 func (w *ChangesWorker) updatePollingInterval(updateType string) {
 
 	if updateType == UPDATE_TYPE_RESET {
-		// set the polling interval to minial. This is the case where agreement negotiation started when the node needs to
+		// set the polling interval to minimal. This is the case where agreement negotiation started when the node needs to
 		// watch the upcoming messages more closely.
 		if w.pollInterval != w.pollMinInterval {
 			w.pollInterval = w.pollMinInterval
 			w.SetNoWorkInterval(w.pollInterval)
 			glog.V(3).Infof(chglog(fmt.Sprintf("Resetting poll interval to %v, max interval is %v, increment is %v.", w.pollInterval, w.pollMaxInterval, w.pollAdjustment)))
 		}
-
 		w.noMsgCount = 0
 	} else if updateType == UPDATE_TYPE_ALERT {
 		// This is the case when the node should be watching the changes more often than max but not as frequent as min.
 		// It is called alert.
-		// For excample: node policy changed, node userinput changed, node itself changed, service changed,
+		// For example: node policy changed, node userinput changed, node itself changed, service changed,
```

Comment:
Please update the comments here and for UPDATE_TYPE_RESET to reflect this change

Reply:
Good suggestion - I updated the comments.